okhttp/build.gradle

```diff
@@ -5,12 +5,20 @@ plugins {
 description = "gRPC: OkHttp"
 dependencies {
     compile project(':grpc-core'),
-            libraries.okhttp
+            libraries.okhttp,
+            libraries.okio
 
     // Tests depend on base class defined by core module.
     testCompile project(':grpc-core').sourceSets.test.output
 }
 
+// TODO(madongfly) Fix the code and remove these exceptions.
```

Comment:
We probably don't want to "fix" the code, as that just forks us further from OkHttp; we want to have as few changes as possible.